content/metadata.yaml

```
@@ -411,12 +411,13 @@ author_info:
 #   twitter: null
 #   affiliations:
 #   - University of California, Berkeley
-# - name: Lawrence Hunter
-#   orcid: null
-#   github: null
-#   twitter: null
-#   affiliations:
-#   - University of Colorado
+- name: Lawrence Hunter
+   orcid: 0000-0003-1455-3370
```

The build may have failed because of 3 instead of 2 leading spaces in lines 415 and below .  E.g.,
```suggestion
  orcid: 0000-0003-1455-3370
```